ripple/src/main/webapp/businessdetails.js

```diff
@@ -6,54 +6,46 @@ $(document).ready(() => {
   });
 });
 
-$(".selector").on("mouseover", function () {
-    //stuff to do on mouseover
-});
-
-/* Set user's star rating in local storage upon click of star.
-   Change color of the stars accordingly  */
-function starRating(starRating) {
-  localStorage.setItem("starRating", starRating);
-  console.log("starRating() called");
-  console.log("starRating local storage: " + localStorage.getItem("starRating"));
+/* Change color of the elements according to user input onmouseover or onclick 
+   If event is onclick, set user choice in local storage.
+   If onclick event is for a star rating, enable the button */
+function addColorToRating(ratingType, rating, maxRating, elementIdPrefix, addClass, isOnClick=false) {
+  if (isOnClick == true) {
+    localStorage.setItem(ratingType, rating);
+    console.log("addColorToRating local storage: " + localStorage.getItem(ratingType));
+    // Enable post button if user has made a star rating
+    if (ratingType == "starRating") {
```

Comment:
nit: probably good practice to have a "starRating" variable you reference here

Reply:
Resolved. Will set a variable if "starRating" is referenced multiple times.